src/web/src/components/ScheduleEvent.vue

```diff
@@ -1,8 +1,17 @@
 <template functional>
   <div class="schedule-event" :style="data.style">
     <div class="event-text">
-      {{ props.crn }} - {{ props.section }}
+      {{ props.title }}
       <br>
+      {{ props.name }} - {{ props.section }}
+      <br>
+      {{ props.crn }}
+      <br>
+      Location
```

Comment:
this part should get the actual course location if it is available

Reply:
Removed until data is available